libcontainer/cgroups/ebpf/devicefilter/devicefilter_test.go

```diff
@@ -26,7 +26,7 @@ func testDeviceFilter(t testing.TB, devices []*devices.Rule, expectedStr string)
 		t.Fatalf("%s: %v (devices: %+v)", t.Name(), err, devices)
 	}
 	s := insts.String()
-	t.Logf("%s: devices: %+v\n%s", t.Name(), devices, s)
+	// t.Logf("%s: devices: %+v\n%s", t.Name(), devices, s)
```

Comment:
Any reason to keep it commented vs. removing it?

Reply:
Not really. Removed.